### PR TITLE
Update room.md

### PR DESCRIPTION
**Pull request by su-chang**
Confirm the return type of instance/static function

### CODE REVIEW

File: root/api/room.md [hunk 1]
```diff
@@ -120,10 +120,10 @@ const linkPayload = new UrlLink({
 await room.say(linkPayload)
 ```
 
-### room.on\(event, listener\) ⇒ `this`
+### room.on\(event, listener\) ⇒ `Room`
 
 **Kind**: instance method of [`Room`](room.md#Room)  
-**Returns**: `this` - - this for chain
+**Returns**: `Room` - - Room for chain
```
Comment (windmemory) [inline root/api/room.md]: I think it would be better to keep the return as `this`.

**Comment by lijiarui** [comment]
Please resolve all of the conversations.

File: root/api/room.md [hunk 2]
```diff
@@ -39,27 +39,27 @@ All wechat rooms\(groups\) will be encapsulated as a Room.
...
+    * [.memberAll\(\[query\]\)](room.md#Room+memberAll) ⇒ `Promise <Contact []>`
+    * [.member\(queryArg\)](room.md#Room+member) ⇒ `Promise <Contact | null>`
     * [.owner\(\)](room.md#Room+owner) ⇒ `Contact` \| `null`
-    * [.avatar\(\)](room.md#room-owner-contact-or-null) ⇒ `Promise.`
+    * [.avatar\(\)](room.md#room-owner-contact-or-null) ⇒ `Promise <FileBox>`
   * _static_
-    * [.create\(contactList, \[topic\]\)](room.md#Room.create) ⇒ [`Promise.`](room.md#Room)
-    * [.findAll\(\[query\]\)](room.md#Room.findAll) ⇒ `Promise.>`
-    * [.find\(query\)](room.md#Room.find) ⇒ `Promise.`
+    * [.create\(contactList, \[topic\]\)](room.md#Room.create) ⇒ [`Promise <Room>`](room.md#Room)
```
[the remark this answers is not in the document]
Comment (su-chang) [reply root/api/room.md]: it's a link for markdown.

File: root/api/room.md [hunk 3]
```diff
@@ -39,27 +39,27 @@ All wechat rooms\(groups\) will be encapsulated as a Room.
...
-    * [.quit\(\)](room.md#Room+quit) ⇒ `Promise.`
-    * [.topic\(\[newTopic\]\)](room.md#Room+topic) ⇒ `Promise.`
-    * [.announce\(\[text\]\)](room.md#Room+announce) ⇒ `Promise.`
-    * [.qrcode\(\)](room.md#Room+qrcode) ⇒ `Promise.`
-    * [.alias\(contact\)](room.md#Room+alias) ⇒ `Promise.`
-    * [.has\(contact\)](room.md#Room+has) ⇒ `Promise.`
-    * [.memberAll\(\[query\]\)](room.md#Room+memberAll) ⇒ `Promise.>`
-    * [.member\(queryArg\)](room.md#Room+member) ⇒ `Promise.`
+    * [.sync\(\)](room.md#Room+sync) ⇒ `Promise <void>`
+    * [.say\(textOrContactOrFileOrUrl, \[mention\]\)](room.md#Room+say) ⇒ `Promise <void>`
```
[the remark this answers is not in the document]
Comment (su-chang) [reply root/api/room.md]: Done, updated!

File: root/api/room.md (hunk 1 above)
Comment (su-chang) [reply root/api/room.md]: OK, agree. `this` for chain is better than `Room` for chain.

File: root/api/room.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (windmemory) [reply root/api/room.md]: I don't see the necessity here. Why keep this?

[the remark this answers is not in the document]
Comment (su-chang) [reply root/api/room.md]: Understand, done.

File: root/api/room.md (hunk 1 above)
Comment (windmemory) [reply root/api/room.md]: Have you updated this?

Comment (su-chang) [reply root/api/room.md]: yes

**Pull request merged**